api/v1alpha1/ocivalidator_types.go

```diff
@@ -70,6 +76,16 @@ type Auth struct {
 	SecretName string `json:"secretName" yaml:"secretName"`
```

Comment:
It would be useful to mention here that the auth secret referred to by SecretName must exist in the same namespace as the OCIValidator.